src/Audio/CueData.cs

```diff
@@ -789,12 +878,23 @@ public uint Timestamp
 			private set;
 		}
 
-		protected static Random random = new Random();
+		public int Count { get; private set; }
```

Comment:
We may have to write it somewhere else since this data is technically supposed to be read-only, but the idea definitely works!